### DISCUSSION TITLE
Managing config patches

### DISCUSSION THREAD

**Discussion by Hr46ph** [siderolabs/talos#7390] [Q&A]
I am wondering how other people manage their config patches. I think this can get out of hand quite easily. Even as a sole OPS engineer if you're not managing this it's easy to loose track of which patches are done in which order, what's current, etc. Imagine a team of 5 or more ... 

I am leaning towards Ansible, is anyone using that for this purpose?

I think with Ansible and group_vars / host_vars it would be easy to generate yaml config patches and even apply them, or gather running configs.

How are others doing this?

**Comment by smira** [comment]
This can be done by wrapping the `talosctl` and some shell scripts to pick up and find appropriate patches for each machine. Also Talos machinery Go module can be used to have any other kind of integration.

You can also look into our higher-level orchestration project built on top of Talos: [Omni](https://www.siderolabs.com/platform/saas-for-kubernetes/)
